src/EVault/modules/Initialize.sol

```diff
@@ -20,6 +20,7 @@ abstract contract InitializeModule is IInitialize, Base, BorrowUtils {
 
     uint256 internal constant INITIAL_INTEREST_ACCUMULATOR = 1e27; // 1 ray
     uint16 internal constant DEFAULT_INTEREST_FEE = 0.1e4;
+    uint16 internal constant DEFAULT_MAX_LIQUIDATION_DISCOUNT = 0.2e4;
```

Comment:
@euler-mab suggested it's best to default this to 0, and force vault creators to choose a value for it. This would mean that if a deployer forgets to do this, liquidations would be non-functional.

I'm still not sure about this. On the one hand, we avoid blessing the arbitrary value of `0.2` and limit our 'liability' by forcing users to choose this on their own. OTOH, this increases the chance that uninformed users will choose an obviously bad value (such as 0).

Reply:
went ahead and removed the default. can add it back